sdk/key.go

```diff
@@ -28,7 +28,7 @@ func Key(keyReq KeyRequest) ([]byte, error) {
 	var capeKey, err = readCapeKey(keyReq.CapeKeyFile)
 	if err != nil {
 		// If the key file isn't present we download it, but log this error anyway in case something else happened.
-		log.Debug("Unable to open cape key file: %w", err)
+		log.Debugf("Unable to open cape key file: %s", err)
```

Comment:
using %w gives weird output
```
Unable to open cape key file: %!w(*fs.PathError=&{open /home/ezphlow/.config/cape/capekey.pub.der 2})
```

Reply:
The `%w` verb is specific to `fmt.Errorf()` for wrapping errors in other error types. Logrus ultimately delegates this to `fmt.Sprintf()` which doesn't support `%w`.

If we're using a structured log implementation like logrus, it's worth considering using `log.WithError(err).Debug("unable to open key file")` instead of formatting like this. The default formatter outputs some fairly readable messages that are easy to grep. We could more easily provide a flag to use the json formatter, which would make these logs easier to index and use in automated systems.

Reply:
I don't think WithError works with Debug right now. Only displays 
```
Unable to open cape key file
```
Do you think it's okay to merge this as is?

Reply:
Looks like that's because we're overriding the default formatter [here](https://github.com/capeprivacy/cli/blob/6c66a08648032b37deb4b4a68d64f542ddce34a8/cmd/cape/cmd/root.go#L42). Merging this as-is won't cause any issues.